Hymns/Audio Player/AudioTimeUtil.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import AVFoundation
+
+class Utility: NSObject {
```

Comment:
Having a generic utility classes is usually antipattern http://ralin.io/blog/oop-anti-patterns-utility-or-helper-classes.html

Reply:
Removed this file in 155b533